src/graph.js

```diff
@@ -40,35 +40,95 @@ class Graph {
   // Optionally accepts an array of other GraphNodes for the new vertex to be connected to
   // Returns the newly-added vertex
   addVertex(value, edges = []) {
-
+    const newVertex = new GraphNode(value, edges);
+    this.vertices.push(newVertex);
+    if (this.vertices.length === 2) {
+      this.addEdge(this.vertices[0], this.vertices[1]);
+    }
+    if (edges.length !== 0 && this.vertices.length > 2) {
+      newVertex.edges.forEach((vertex) => {
+        this.addEdge(newVertex, vertex);
+      });
+    }
+    return newVertex;
   }
   // Checks all the vertices of the graph for the target value
   // Returns true or false
   contains(value) {
-
+    let found = false;
+    this.vertices.forEach((vertex) => {
+      if (value === vertex.value) {
+        found = true;
```

Comment:
Once the `found` flag has been flipped, you can `break` out of this loop to just end this function's execution, instead of going on iterating through the rest of the elements in the array. There's really no need to.